1_OCaml/TP/TP4.ipynb

```diff
@@ -19,7 +19,9 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.                    let listinter = l::(parties q acc) in 
</pre><p>Ce n'est pas très clair et il faudrait utiliser add défini juste avant</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mp2i-fsm/mp2i-2021/pull/110/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>